### PR TITLE
v5 docs part 1

### PR DESCRIPTION
**Pull request by jasonblais**
1. Replaces platform binary with Mattermost: hopefully captured everything
2. Upgrade notes: https://github.com/mattermost/docs/pull/2039/commits/460fe8949d984403ec36277255ffbe5f4578f533 and https://github.com/mattermost/docs/pull/2039/commits/5e326968db233ece7215b57f3711c7907d6bea20
3. Minor APIv3 —> APIv4 update https://github.com/mattermost/docs/pull/2039/commits/fd818460af276aa75482b3cc6fa600b675cec8f3

We should get at least two devs to review.

### CODE REVIEW

File: source/install/smtp-email-setup.rst [hunk 1]
```diff
@@ -67,7 +67,7 @@ To enable email, configure an SMTP email service as follows:
       9. Then click **Save**
       10. Then click **Test Connection**
       11. If the test failed please look in **OTHER** > **Logs** for any
-          errors that look like ``[EROR] /api/v3/admin/test_email ...``
+          errors that look like ``[EROR] /api/v4/email/test ...``
```
Comment (jasonblais) [inline source/install/smtp-email-setup.rst]: Updated to v4 endpoint

**Comment by jasonblais** [comment]
@crspeller It's there (although we may not do that afterall) https://github.com/mattermost/docs/pull/2039/files#diff-5ac0fed59dd5ab36847efd3badee7cc9R15

Install guide already requires it.

File: source/administration/important-upgrade-notes.rst [hunk 2]
```diff
@@ -2,11 +2,33 @@ Important Upgrade Notes
...
+|                                                    | A new ``config.json`` setting to disable the `permanent APIv4 delete team parameter                                                                             |
+|                                                    | <https://api.mattermost.com/#tag/teams%2Fpaths%2F~1teams~1%7Bteam_id%7D%2Fput>`_ is added. The setting will be off by default for all new and existing          |
+|                                                    | installs, except those deployed on GitLab Omnibus. If you reply on the APIv4 parameter, enable the setting in `config.json` before upgrading.                   |
+|                                                    +-----------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | An unused ``ExtraUpdateAt`` field will be removed from the channel model.                                                                                       |
```
Comment (esethna) [inline source/administration/important-upgrade-notes.rst]: Does this need to be in the upgrade notes if it's an unused field? I may be missing some context here.

Comment (jasonblais) [reply source/administration/important-upgrade-notes.rst]: It wasn't used by the software but someone might have used it via an API

**Pull request merged**